README.md

```diff
@@ -96,6 +96,46 @@ Setting up the server
 		uwsgi_pass ...;
 	}
 
+Setting up the server (Apache version)
+---------------------
+
+If you wish to use Apache to host the master server, do steps 1-2, 4, above. Additionally install/enable mod_wsgi and an Apache site config like the following:
+
+	<VirtualHost *:80>
+		ServerName servers.minetest.net
+		ServerAlias servers.minetest.net
+		ServerAdmin admin@local.server
```

Comment:
This section is already quite large so I suggest leaving off the `<VirtualHost>` and these three lines since they don't directly relate to to wsgi configuration.